docs/language-server/protocol-language-server.md

```diff
@@ -545,6 +546,26 @@ interface TextEdit {
 }
 ```
 
+### `ErrorLocation`
```

Comment:
I think this should be made more extensible. It's not just about _errors_ but about diagnostics in general, so I would suggest `DiagnosticLocation` with a `DiagnosticType = Error | Warning | Lint` field.